crates/loro-internal/src/state.rs

```diff
@@ -38,14 +38,25 @@ pub(crate) use tree_state::{get_meta_value, TreeState};
 
 use super::{arena::SharedArena, event::InternalDocDiff};
 
+macro_rules! get_or_create {
```

Comment:
Is it necessary to use macros?

Reply:
Yes, otherwise there are lifetime issues